dvach-browser/VLCPlayerViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  VLCPlayerViewController.swift
+//  dvach-browser
+//
+//  Created by Victor Cebanu on 2/3/20.
+//  Copyright © 2020 8of. All rights reserved.
+//
+
+import Foundation
+
+final class VLCPlayerViewController: UIViewController, VLCPlayerViewInput {
+    var interactor: VLCPlayerViewOutput!
```

Comment:
Let's use dependency injection.
Make our own `init` and set `interactor` inside.

Reply:
we inject the view controller into the interactor
we can't do it both ways
check `VLCPlayerScreenBuilder.swift`